plugins/modules/volume_cloudinit.py

```diff
@@ -126,11 +131,14 @@ def cloud_localds(volume_format,
                   ci_networkconfig_path,
                   configdrive_path,
                   module):
-    # TODO: Reimplement cloud-localds in Python and drop dependency on cloud-localds,
+    # Old: Reimplement cloud-localds in Python and drop dependency on cloud-localds,
     #       because it is not available on Red Hat Enterprise Linux 8 / 9 and CentOS 8 / 9.
     #       Ref.: https://salsa.debian.org/cloud-team/cloud-utils/-/blob/master/bin/cloud-localds
 
-    cmd = 'cloud-localds'
+    cmd = 'cloud-localds' if module.params['kvm_host_os'] != 'Red Hat Enterprise Linux' else '/usr/local/cloud-localds'
+    module.debug(module.params['kvm_host_os'])
+    module._log_to_syslog
+    cmd = '/usr/local/cloud-localds'
```

Comment:
here is where i check the `kvm_host_os` but i forgot to remove the module debug command